cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/release.go

```diff
@@ -249,17 +255,42 @@ func (s *Server) installedPackageDetail(ctx context.Context, name types.Namespac
 		ValuesApplied:         valuesApplied,
 		ReconciliationOptions: installedPackageReconciliationOptionsFromUnstructured(unstructuredRelease.Object),
 		AvailablePackageRef:   availablePackageRef,
-		// TODO (gfichtenholt) PostInstallationNotes
-		// looks like I may have to resort to getting the notes via helm APIs, flux doesn't do it
-		// see discussion in https://cloud-native.slack.com/archives/CLAJ40HV3/p1629244025187100
-		// bottom line: "helm get notes" is a read-only activity that you can replicate in your
-		// own code by copying the implementation, or calling to the library function if there is one
-		// https://github.com/helm/helm/blob/main/cmd/helm/get_notes.go.
-		// Also look at https://gist.github.com/DzeryCZ/c4adf39d4a1a99ae6e594a183628eaee
-		Status: installedPackageStatusFromUnstructured(unstructuredRelease.Object),
+		PostInstallationNotes: postInstallNotes,
+		Status:                installedPackageStatusFromUnstructured(unstructuredRelease.Object),
 	}, nil
 }
 
+func (s *Server) postInstallationNotesFromUnstructured(ctx context.Context, name types.NamespacedName, unstructuredRelease map[string]interface{}) (string, error) {
+	// post installation notes can only be retrieved via helm APIs, flux doesn't do it
+	// see discussion in https://cloud-native.slack.com/archives/CLAJ40HV3/p1629244025187100
+	if s.actionConfigGetter == nil {
+		return "", status.Errorf(codes.FailedPrecondition, "Server is not configured with actionConfigGetter")
+	}
+
+	helmReleaseName, found, err := unstructured.NestedString(unstructuredRelease, "spec", "ReleaseName")
+	// according to docs ReleaseName is optional and defaults to a composition of
+	// '[TargetNamespace-]Name'.
+	if !found || err != nil || helmReleaseName == "" {
+		targetNamespace, found, err := unstructured.NestedString(unstructuredRelease, "spec", "targetNamespace")
+		// according to docs targetNamespace is optional and defaults to the namespace of the HelmRelease
+		if !found || err != nil || targetNamespace == "" {
+			targetNamespace = name.Namespace
+		}
+		helmReleaseName = fmt.Sprintf("%s-%s", targetNamespace, name.Name)
```

Comment:
As above, I don't see why we'd need to construct the release name, unless you've found that what I mention above is incorrect and a `HelmRelease` CRD can exist without a `ReleaseName`, but that would be very odd?